src/test/java/com/mael/payroll/cards/TimeCardTest.java

```diff
@@ -7,8 +7,8 @@
 public class TimeCardTest {
     @Test
     public void createsATimeCard() {
-        SalesCard salesCard = new SalesCard(10001010, 8.0);
-        assertEquals(10001010, salesCard.getDate());
-        assertEquals(8.0, salesCard.getAmount(), 0.001);
+        TimeCard timeCard = new TimeCard(10001010, 8.0);
```

Comment:
Check [this](https://github.com/me-stevens/payroll/pull/3#discussion_r48946949) out.